style.css

```diff
@@ -234,12 +243,111 @@
     align-items: center;
   }
 
-  .grid-3-movile {
+  .projectCards {
     position: relative;
     width: 100%;
     top: 139px;
   }
 
+  .contPopUp {
+    z-index: 10;
+    display: none;
+    position: fixed;
+    padding: 10%;
+    color: black;
+    width: 100vw;
+    height: 100vh;
+    box-sizing: border-box;
+    backdrop-filter: blur(5px);
+  }
+
+  .popUp-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  .contentPopUp {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    width: 100%;
+    height: 100vh;
+    padding: 0% 5% 5% 5%;
+    box-sizing: border-box;
+  }
+
+  .popUp-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+  }
+
+  .popUpImg {
+    width: 100%;
+    height: auto;
+  }
+
+  .exitButton {
+    left: 150px;
+    background-color: transparent;
+    background-image: url(images/exit.png);
+    width: 24px;
+    height: 24px;
+    border: none;
+  }
+
+  .popUp-stack {
+    padding: 0%;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-align: center;
+  }
+
+  .popUp-btns {
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    position: relative;
+  }
+
+  .button-Live {
+    font-size: 17px;
+    color: white;
+    background-color: #ff6b00;
+    padding: 10px 12px;
+    width: 129px;
+    height: 48px;
+    border: 1px solid #ff6b00;
+    text-align: start;
+  }
+
+  .button-Source {
+    font-size: 17px;
+    color: white;
+    background-color: #ff6b00;
+    padding: 10px 12px;
+    width: 158px;
+    height: 48px;
+    border: 1px solid #ff6b00;
+    text-align: start;
+  }
+
+  .img-btn {
+    z-index: 200;
+    width: 30px;
+    height: auto;
+    position: relative;
+    right: 35px;
+  }
+
```

Comment:
Great job implementing the changes by the previous reviewer, however as suggested  by the previous reviewer, the mobile version currently has an overflow which is as a result of using excess height for the popup hence it doesn't display the full mobile popup. please effect this change so as to have a better interface 😄 

Your design :
![image](https://github.com/Zven94/portfolio/assets/20537712/15332b41-cbed-4fc6-99a4-9a9f5130aa36)

Figma design :
![image](https://github.com/Zven94/portfolio/assets/20537712/073ac7a2-5dea-4b5a-bd8a-947cd13bda32)

Reply:
Hi! Thank you for your time. You 're right I forgot to update the change into git.
I'm doing it now. Thank you 💪